src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -3283,17 +3283,24 @@ bool ApplicationManagerImpl::HMILevelAllowsStreaming(
     uint32_t app_id, protocol_handler::ServiceType service_type) const {
```

Comment:
This function should probably be renamed since it now checks both the hmi level and the streaming state. Maybe `HMIStateAllowsStreaming` would work better

Reply:
@ShobhitAd fixed in db2d1c86722705169c2633a18b01c6f072059952